volatility/framework/plugins/linux/tty_check.py

```diff
@@ -0,0 +1,77 @@
+# This file is Copyright 2020 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+from typing import List
+
+from volatility.framework import interfaces, renderers, exceptions, constants, contexts
+from volatility.framework.automagic import linux
```

Comment:
Merge request #266 is probably going to go in first (as agree with @atcuno at the meeting on Wednesday), so this will need to be changed from `automagic` to `mac` but no other changes should be required.

Reply:
Looks like it's just a small conflict over the imports.  My last attempts at using the web UI to fix the conflicts failed, so I'd prefer if you'd be able to rebase and force push this back to your branch, then we can get this merged (depending on the above designs on elf parsing).

Reply:
I'm getting some help in making sure I properly rebase it on my end then I will recommit the new changes

Reply:
Awesome, thanks!  5:)